core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/variable/listener/support/VariableListenerNotifiable.java

```diff
@@ -20,16 +20,22 @@
 import java.util.Collection;
 
 import org.optaplanner.core.api.domain.variable.VariableListener;
+import org.optaplanner.core.api.score.director.ScoreDirector;
 
 final class VariableListenerNotifiable<Solution_> implements Comparable<VariableListenerNotifiable<Solution_>> {
 
-    private final VariableListener<Solution_, ?> variableListener;
+    private final ScoreDirector<Solution_> scoreDirector;
+    private final VariableListener<Solution_, Object> variableListener;
     private final int globalOrder;
 
     private final Collection<VariableListenerNotification> notificationQueue;
 
-    public VariableListenerNotifiable(VariableListener<Solution_, ?> variableListener, int globalOrder) {
-        this.variableListener = variableListener;
+    VariableListenerNotifiable(
+            ScoreDirector<Solution_> scoreDirector,
+            VariableListener<Solution_, ?> variableListener,
+            int globalOrder) {
```

Comment:
I think this is incorrect formatting.

Reply:
Not according to the Quarkus style. Feel free to suggest an improvement.